tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -101,24 +92,20 @@ before_break2 = (
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
     />
-    {" "}
-    foo
+    {" "}foo
   </span>
 );
 
 after_break = (
   <span>
     foo
-    {" "}
-    <span barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar />
+    {" "}<span
```

Comment:
I have a PR that switches `{" "}` positioning around so they now appear at the end of lines rather than the beginning. I'd be interested in your thoughts.

https://github.com/prettier/prettier/pull/1964